configure

```diff
@@ -784,6 +784,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
For what I know, autoconf is now current at 2.69, and have no plan for new version release[1].

[1]: https://lists.gnu.org/archive/html/autoconf/2016-08/msg00014.html